src/web/routes/application/check/post.test.js

```diff
@@ -81,14 +81,20 @@ test(`successful post sets next allowed step to ${CONFIRM_URL} and eligibility s
 
   post.returns(Promise.resolve({
     body: {
-      eligibilityStatus: ELIGIBLE
+      eligibilityStatus: ELIGIBLE,
```

Comment:
shouldn't we also include claimStatus: ACTIVE here?

Reply:
The claimStatus isn't added to the session so isn't needed here.